src/Http/Response.php

```diff
@@ -206,13 +175,9 @@ public static function file(string $file, array $props = [])
 	 * Urls can be relative or absolute.
 	 * @since 3.7.0
 	 *
-	 * @param string $url
-	 * @param int $code
-	 * @return void
-	 *
 	 * @codeCoverageIgnore
 	 */
-	public static function go(string $url = '/', int $code = 302)
+	public static function go(string $url = '/', int $code = 302): void
```

Comment:
We could add a `TODO` comment to change the return type to `never` once we drop support for PHP 8.0.